web/templates/message_body.hbs

```diff
@@ -26,8 +26,6 @@
     {{/if}}
 </span>
 
-<span class="alert-msg"></span>
```

Comment:
It's nice to be able to pull this unused element out of every message row!

Reply:
Seriously! I need to open up another PR that put the message-row elements in a more sensible order, too. Like, the timestamp appears ahead of the hover controls.